granary/reddit.py

```diff
@@ -0,0 +1,215 @@
+# coding=utf-8
+"""reddit.com source class.
+"""
+
+from . import source
+import datetime
+import logging
+from oauth_dropins import reddit
+from oauth_dropins.webutil import util
+import re
+import urllib.parse, urllib.request
+
+import praw
+
+class Reddit(source.Source):
+
+    DOMAIN = 'reddit.com'
```

Comment:
sadly granary (and bridgy and oauth-dropins) are non-PEP8 and use 2 column indents instead of 4 column. mind updating this file?